README.md

```diff
@@ -3,45 +3,223 @@
+```
+
+Check out list of available NOAA BDP data sets here at [this link](
+https://www.noaa.gov/organization/information-technology/list-of-big-data-program-datasets)
+
+When you are browsing the AWS buckets, you can find the appropriate file path to
+use with `wget` by "right clicking" the file name, and copying the link address. 
+
 ## Run the package
 
-An example script has been included for generating a single figure, and serves
-as an example for how to call the package for an upper air map. To run the
-example script in the top-level repo directory, type:
+### Creating maps
+
+#### Configure the list of fields
+
+The real-time graphics produce more than 100 maps for each model at each
+forecast lead time. The list of maps is configured in the pygraf subdirectory
+`image_lists/` where you will find a set of yaml files for a variety of
+supported NWP systems run at GSL.
+
+Start with the one that matches the model you'd like to plot, and remove or
+comment (add a `#` at the beginning of the line) out the fields you are not
+interested in.
+
+You will provide this file path when you run the graphics in the next step.
+
+#### Submitting the run script
+
+See a full list of command line arguments by running the following command:
 
 ```
-conda activate pygraf
-python plot_example.py
+python create_graphics.py -h
+```
+
+If you are creating only a couple of maps using RRFS data as an example you can
+run on the front-end nodes (or on your laptop) with a command like this:
+
+```
+python create_graphics.py \
+         maps \
+         --all_leads \
+         -d /path/to/input/data \
+         -f 0 6 \
+         --file_type prs \
+         --file_tmpl "RRFS_NA_3km.t15z.bgdawpf{FCST_TIME:03d}.tm00.grib2" \
+         --images ./image_lists/rrfs_subset.yml hourly \
+         -m "My RRFS Retro" \
+         -n 4 \
+         -o /path/to/output/images \
+         -s 2021052315 \
+         --tiles full
+```
+
+If you have a larger set of maps to create, you might use a Slurm batch script
+that looks like this, and is submitted from the pygraf directory:
+
+```
+#!/bin/bash
+
+#SBATCH --account=my_jet_account
+#SBATCH --qos=batch
+#SBATCH --nodes=1-1
+#SBATCH --exclusive
+#SBATCH --partition=kjet,xjet,tjet,ujet
+#SBATCH -t 1:30:00
+#SBATCH --job-name=maps
+
+source pre.sh
+
+python create_graphics.py \
+         maps \
+         --all_leads \
+         -d /path/to/input/data \
+         -f 0 12 \
+         --file_type prs \
+         --file_tmpl "RRFS_NA_3km.t15z.bgdawpf{FCST_TIME:03d}.tm00.grib2" \
+         --images ./image_lists/rrfs_subset.yml hourly \
+         -m "My RRFS Retro" \
+         -n ${SLURM_CPUS_ON_NODE:-12} \
+         -o /path/to/output/images \
+         -s 2021052315 \
+         --tiles full,ATL,CA-NV,CentralCA
+
+```
+NOTE: The graphics already run as a workflow step in the RRFS Retros! They may be
+zipped by default, so you can unzip those files to see your images on disk.
```

Comment:
Should we note here that the zipped files get moved over from `pyprd` to `nclprd`?  I remember Eric learned that the hard way. ;-)